acceptance-tests/upgrade/update_and_upgrade_mssql_failover_group_existing_test.go

```diff
@@ -0,0 +1,124 @@
+package upgrade_test
+
+import (
+	"csbbrokerpakazure/acceptance-tests/helpers/apps"
+	"csbbrokerpakazure/acceptance-tests/helpers/brokers"
+	"csbbrokerpakazure/acceptance-tests/helpers/random"
+	"csbbrokerpakazure/acceptance-tests/helpers/services"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Upgrade and Update csb-azure-mssql-db-failover-group 'existing' plan", Label("mssql-failover-group-existing"), func() {
+	When("upgrading broker version", Label("modern"), func() {
+		FIt("should continue to work", func() {
```

Comment:
Hey @blgm I just removed the focus from this line. Git pull if you have to change something